plugins/woocommerce/README.md

```diff
@@ -12,22 +12,27 @@ If you are not a developer, please use the [WooCommerce plugin page](https://wor
 
 ## Getting Started
 
-Please make sure you follow the [repository's getting started guide](../../README.md#getting-started) first!
+### Prerequisites
+
+-   [Repository "Getting Started" Guide](../../README.md#getting-started)
+-   [Docker](https://docs.docker.com/get-docker/)
+
+Once you have completed the prerequisites, you can start the development environment:
 
 ```bash
-# Make sure that WooCommerce Core and all of its dependencies are built
-pnpm --filter=@woocommerce/plugin-woocommerce build 
-# Make sure you're in the WooCommerce Core directory
-cd plugins/woocommerce
+## Watch for changes in WooCommerce and all of its dependencies.
+pnpm --filter='@woocommerce/plugin-woocommerce' watch:build
```

Comment:
The reason for the comment above (re: Quick start) is that this is the only development command most people will need.